src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,13 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length > 0
```

Comment:
Don't check it here. Move it to PostInfo component and render CommentList or "no comments" message there